### PR TITLE
SAT filter-bin cleanup 

### PR DESCRIPTION
**Pull request by susannaaz**

### CODE REVIEW

File: sotodlib/site_pipeline/make_filterbin_map.py [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 from scipy import ndimage, interpolate
 from scipy.optimize import curve_fit
 from scipy.stats import kurtosis, skew
-#from tqdm import tqdm
 from . import util
+from sotodlib.tod_ops.fft_ops import calc_psd, calc_wn
+from sotodlib.hwp import hwp_angle_model
+from scipy.signal import welch
+from sotodlib import core
+from pixell import utils as pixut
```
Comment (chervias) [inline sotodlib/site_pipeline/make_filterbin_map.py:20]: This line is not needed

File: sotodlib/site_pipeline/make_filterbin_map.py [hunk 2]
```diff
@@ -148,508 +175,519 @@ def find_footprint(context, tods, ref_wcs, comm=mpi.COMM_WORLD, return_pixboxes=
...
-        ivar = 1.0/np.var(obs.demodQ, axis=-1)
-        mask_det = ivar > np.percentile(ivar, 95)
-        obs.restrict('dets', obs.dets.vals[~mask_det])
-    return obs
 
-def calibrate_obs_otf(obs, dtype_tod=np.float32, site='so_sat1'):
+def ptp_cuts(obs, signal_name='dsT'): #, kurtosis_threshold=5):
+    # Find the typical noise level in the filtered tod, ignoring glitches
+    bsize  = 100
+    rms  = np.percentile(pixut.block_reduce(obs.signal, bsize,
```
Comment (chervias) [inline sotodlib/site_pipeline/make_filterbin_map.py:201]: replace pixut with utils here

**Review by chervias — APPROVED** [review]
Ok, there is one detail left to fix, after that merge please

File: sotodlib/site_pipeline/make_filterbin_map.py (hunk 1 above)
Comment (susannaaz) [reply sotodlib/site_pipeline/make_filterbin_map.py]: Done

File: sotodlib/site_pipeline/make_filterbin_map.py (hunk 2 above)
Comment (susannaaz) [reply sotodlib/site_pipeline/make_filterbin_map.py]: Done

**Pull request merged**